src/main/java/laddergame/domain/GameResult.java

```diff
@@ -0,0 +1,46 @@
+package laddergame.domain;
+
+import static laddergame.utils.ErrorMessage.INVALID_GAME_RESULT_LENGTH;
+
+public class GameResult {
+    private final static int MIN_RESULT_LENGTH = 1;
+    private final static int MAX_RESULT_LENGTH = 5;
+    private final static String BLANK = " ";
+
+    private final String result;
+
+    public GameResult(String result) {
+        validateResult(result);
+        this.result = result;
+    }
+
+    private void validateResult(String result) {
+        validateResultLength(result);
+        validateResultFormat(result);
+    }
+
+    private void validateResultLength(String result) {
+        if (isLongerThanMaxLength(result) || isShorterThanMinLength(result)) {
+            throw new IllegalArgumentException(
+                    String.format(INVALID_GAME_RESULT_LENGTH.getMessage(), MIN_RESULT_LENGTH, MAX_RESULT_LENGTH));
+        }
+    }
+
+    private boolean isShorterThanMinLength(String result) {
+        return result.length() < MIN_RESULT_LENGTH;
+    }
+
+    private boolean isLongerThanMaxLength(String result) {
+        return result.length() > MAX_RESULT_LENGTH;
+    }
+
+    private void validateResultFormat(String result) {
+        if (result.contains(BLANK)) {
```

Comment:
```suggestion
        if (result.isBlank()) {
```

result가 공백인지 확인하는 로직이 맞을까요?
그렇다면 String 클래스의 isBlank()를 사용할 수도 있어요.

Reply:
+) isNull(), isEmpty(), isBlank()의 차이를 알아두면 좋아요!

Reply:
테스트 코드에서 확인해주셨듯이, 공백 포함 여부를 확인하기 때문에 다른 로직이라고 생각했습니다!!
말씀하신 내용에 대해서도 학습하도록 하겠습니다!! 감사합니다ㅎㅎ

Reply:
말씀하신 3개의 메소드의 차이에 대해 알아보았습니다! 이렇게 코멘트로 증명하는 걸 의도하신 건 아니겠지만.. 남기고 싶었습니다! :)

- isNull() : 참조값이 `null`인지 검사합니다. `null`일 경우 true, 참조하는 객체가 있을 경우 false 반환
    
    ```jsx
    String str = null;
    str.isNull(); -> true 반환
    ```
    
- isEmpty() : 빈 문자열인지 검사합니다. 문자열의 길이가 0이면 true, 1 이상이면 false 반환
    
    ```jsx
    String str = "";
    str.isEmpty(); -> true 반환
    ```
    
- isBlank() : 문자열이 비어있거나, 공백만 있는지 검사합니다. 공백 문자만 있는 경우 true, 그렇지 않은 경우 false 반환
    
    ```jsx
    String str1 = "", str2 = "   ";
    str1.isEmpty(); -> true 반환
    str2.isEmpty(); -> true 반환
    ```

Reply:
코멘트에 기록하는 습관 좋아요 👍
요런 내용을 개인 노션이나 블로그에 기록해두면 나중에 큰 도움이 될 거예요 ㅎㅎ